apps/include/apps.h

```diff
@@ -203,7 +203,7 @@ typedef struct ca_db_st {
 } CA_DB;
 
 void app_bail_out(char *fmt, ...);
-void* app_malloc(int sz, const char *what);
+void* app_malloc(size_t sz, const char *what);
```

Comment:
Could you please move adjust the spaces in the declaration to `void *app_malloc...`